README.md

```diff
@@ -74,6 +74,22 @@ go run make.go
 
 It will build gohai using the `go build` command, with the version info passed through `-ldflags`.
 
+## Development
+
+### Running the code
+
+Ensure that you have Golang installed. You can then run the `gohai` code with:
+
+```sh
+go run gohai.go log.go
+```
```

Comment:
We should have a `go build` command instead I think.

Reply:
We already have that higher in the doc. The problem this change is trying to solve is for quicker test cycles where it's faster to use this line instead of `go build` -> `gohai` workflow.

PS: I'm a bit iffy actually why `log.go` needs to be in toplevel anyways - maybe you know the reason why? Also, I'm curious around why `gohai.go` isn't in something like `cmd/**`?

Reply:
I don't know, this repo is pretty old and never really followed Golang way of doing things.

For testing, would we not simply run the tests ?